test/test_config.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=attribute-defined-outside-init
+from .base import PGHoardTestCase
+from pghoard.config import validate_config_file
+from pghoard.rohmu.errors import InvalidConfigurationError
+from io import StringIO
+import json
+import pytest
+
+
+class TestConfig(PGHoardTestCase):
+
+    def test_validate_config_file(self):
+        raw_config = self.config_template()
+        does_validate = validate_config_file(StringIO(json.dumps(raw_config)))
```

Comment:
As mentioned elsewhere please call this the way it's called from the actual code.